### ISSUE TITLE
not segmenting properly.

### ISSUE THREAD

**Issue by ghost** [bedapudi6788/deepsegment#27]
from deepsegment import DeepSegment
segmenter = DeepSegment('en')
segmenter.segment('I am Batman i live in gotham')

 It gives me this ['I am Batman', 'i live in gotham'] 

but when i give it

from deepsegment import DeepSegment
segmenter = DeepSegment('en')
segmenter.segment('hello what is your name how are you ') 

then it gives me 
 'hello what is your name who are you'

what would you advise?

**Comment by bedapudi6788** [comment]
For smaller sentences, the accuracy suffers because of the low context available. FInetuning or re-training on your type of data will give you the best results. 

Training/ fine-tuning deep segment is easy, and can be done on google colab for free.
